pyproject.toml

```diff
@@ -4,6 +4,22 @@ target_version = ['py310', 'py311', 'py312']
 skip-string-normalization = true
 skip-magic-trailing-comma = true
 
+
+[tool.coverage.run]
+include = [
+    # Omit files outside the current working directory.
+    # Note: this means coverage must be run from the cirq repo root.
+    # Failure to do so will result in false positives.
+    "./*",
+]
+omit = ["benchmarks/*"]
```

Comment:
Would it be worth excluding docs/* as well? It doesn't seem that pytest will do anything with jupyter notebooks without a package like [nbval](https://nbval.readthedocs.io/en/latest/) or [pytest-notebook](https://pytest-notebook.readthedocs.io/en/latest/), and if that's the case, perhaps we may as well skip docs at this time.

Reply:
> Would it be worth excluding docs/* as well? It doesn't seem that pytest will do anything with jupyter notebooks without a package like [nbval](https://nbval.readthedocs.io/en/latest/) or [pytest-notebook](https://pytest-notebook.readthedocs.io/en/latest/), and if that's the case, perhaps we may as well skip docs at this time.

There are indeed no Python sources under the docs directory so adding exclusion for docs would have no effect.
If a setting is unnecessary we should not add it.

Reply:
My point was not clear: it was more about efficiency, to make it skip descending down the paths only to find nothing to do. (Admittedly there are not many files, so the efficiency gain would be trivial.)

But it's okay to leave it off, too.